project/OSGi.scala

```diff
@@ -18,7 +18,8 @@ object OSGi {
     packagedArtifact in (Compile, packageBin) <<= (artifact in (Compile, packageBin), OsgiKeys.bundle).identityMap,
     // This will fail the build instead of accidentally removing classes from the resulting artifact.
     // Each package contained in a project MUST be known to be private or exported, if it's undecided we MUST resolve this
-    OsgiKeys.failOnUndecidedPackage := true 
+    OsgiKeys.failOnUndecidedPackage := true,
+    OsgiKeys.privatePackage := Seq()
```

Comment:
Add a comment here about why. Something like "By default an entry is generated from module group-id, but our modules does not adhere to that such package naming"